### ISSUE TITLE
"Prepare rig for Unity" button not showing in Blender 2.9

### ISSUE THREAD

**Issue by CatDadJynx** [AlexLemminG/Rigify-To-Unity#7]
"Prepare rig for Unity" button not showing in Blender 2.9, verified add on is installed but for some reason isn't showing up.

**Comment by CursedSperk** [comment]
Same

**Comment by AlexLemminG** [comment]
please make sure generated "rig" object is selected and check Object Data Properties tab. Works fine with blender 2.94.3 and Rigify 0.6.3
![изображение](https://user-images.githubusercontent.com/3332929/135333105-e30dbe66-0723-4196-a1a1-f23a20cdb599.png)

**Comment by CatDadJynx** [comment]
Hmm... odd, I did have the generated rig selected but ok.

**Comment by AlexLemminG** [comment]
it should be generated by rigify plugin when you press "Generate Rig" button

**Comment by CatDadJynx** [comment]
Yes, I was trying to use 'generate rig' but was unable to proceed after
doing so because the 'prepare rig for unity' button wasn't showing under
the Object Data Properties tab.

**Comment by AlexLemminG** [comment]
was generated "rig" object selected?

**Comment by CatDadJynx** [comment]
Yes, I made sure it was.

Unfortunately I couldn't afford to wait two months to get it working
(because it was my player character model) so I had to just pay the person
helping me to rig it for me, but I do have other models I'll need to get
working with rigify to unity so I'll try it again though.

And I do know i definitely had the "generated rig" object selected and made
sure I had the rigify add on installed (even uninstalled then reinstalled
it but the Object Data Properties tab still wasn't showing 'prepare rig for
unity' button).
